test/built-ins/Temporal/Calendar/prototype/dayOfYear/plain-date-time.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.dayofyear
+description: Temporal.Calendar.prototype.dayOfYear will take PlainDateTime object and
+return the day of year.
```

Comment:
I think this has to be indented or it's not valid YAML (test262-stream balks at it)